scripts/build-binary

```diff
@@ -59,23 +59,33 @@ function main {
 
 	for os in darwin windows linux; do
 		export GOOS=${os}
-		export GOARCH="amd64"
+		TARGET_ARCHS=("amd64")
 
-		pushd etcd >/dev/null
-			GO_LDFLAGS="-s" ./build
-		popd >/dev/null
-
-		TARGET="etcd-${VER}-${GOOS}-${GOARCH}"
-		mkdir ${TARGET}
-		package ${TARGET} ${PROJ}
-
-		if [ ${GOOS} == "linux" ]; then
-			tar cfz ${TARGET}.tar.gz ${TARGET}
-			echo "Wrote release/${TARGET}.tar.gz"
-		else
-			zip -qr ${TARGET}.zip ${TARGET}
-			echo "Wrote release/${TARGET}.zip"
+		if [[ ${GOOS} == "linux" ]]; then
```

Comment:
this is still `if [[ ${GOOS} == "linux" ]]; then` it should be `if [ ${GOOS} == "linux"]; then` to match the style below